.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,86 +1,48 @@
+          required: true
+        - label: I have searched the [issue tracker](https://github.com/espressif/tinyusb/issues?q=is%3Aissue) for a similar issue and not found any related issue.
+          required: true
   - type: input
+    id: idf_version
     attributes:
-      label: Board
-      placeholder: e.g Feather nRF52840 Express
-    validations:
-      required: true
-
-  - type: textarea
-    attributes:
-      label: Firmware
-      placeholder: |
-        e.g examples/device/cdc_msc. If it is custom firmware, it is preferably compiled like one in example folder and reviewable for people to comment on. The easiest way is
-        - Fork this repo, checkout a new branch
-        - Add your-own-example based on stock one
-        - Push and post it here.
+      label: ESP-IDF version.
+      description: On which ESP-IDF version does this issue occur on? Run `git describe --tags` in your esp-idf folder to find it.
+      placeholder: ex. v5.0-rc1
     validations:
       required: true
-
-  - type: textarea
-    attributes:
-      label: What happened ?
-      placeholder: A clear and concise description of what the bug is.
-    validations:
-      required: true
-
-  - type: textarea
+  - type: input
+    id: devkit
     attributes:
-      label: How to reproduce ?
-      placeholder: |
-        Exact steps in chronological order, details should be specific e.g if you use a command/script to test with, please post it as well.
-        1. Go to '...'
-        2. Click on '....'
-        3. See error
+      label: Development Kit.
+      description: On which Development Kit does this issue occur on?
+      placeholder: ex. ESP32-Wrover-Kit v2 | Custom Board
     validations:
       required: true
-
-  - type: textarea
+  - type: input
+    id: component_version
     attributes:
-      label: Debug Log as txt file (LOG/CFG_TUSB_DEBUG=2)
-      placeholder: |
-        Attach your debug log txt file here, where the issue occurred, best with comments to explain the actual events.
-
-        Note1: Please DO NOT paste your lengthy log contents here since it hurts the readability.
-        Note2: To enable logging, add `LOG=2` to to the make command if building with stock examples or set `CFG_TUSB_DEBUG=2` in your tusb_config.h.
-        More information can be found at [example's readme](https://github.com/hathach/tinyusb/blob/master/docs/getting_started.md)
+      label: Used Component version.
+      description: On which Component version does this issue occur on? Check `dependencies.lock` file in your project root to find it.
+      placeholder: ex. v1.2.0-rc0
     validations:
       required: true
-
   - type: textarea
+    id: more-info
     attributes:
-      label: Screenshots
-      description: If applicable, add screenshots to help explain your problem.
+      label: More Information.
+      description: Do you have any other information from investigating this?
```

Optionally: we can add a word issue, or a bug.. or something similar. Just to be sure  
```suggestion
      description: Do you have any other information from investigating this issue?
```